vhdbuilder/packer/components.json

```diff
@@ -26,7 +26,8 @@
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/exechealthz:*",
       "versions": [
-        "1.2"
+        "1.2",
+        "1.2_v0.0.5"
```

Comment:
what is this used for? fyi this component is retired/EOL upstream https://github.com/kubernetes-retired/contrib/tree/master/exec-healthz

Reply:
"and there should be some issue." what is the issue? and why do we still need it?